### PR TITLE
Create spitter.java

### PR DESCRIPTION
**Pull request by sirswagger21**
Done with a frame for this to-do: https://3.basecamp.com/3361955/buckets/1259843/todos/4533133902

### CODE REVIEW

File: src/main/java/frc/team449/_2022robot/cargo/spitterClass.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package frc.team449._2022robot.cargo;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team449.generalInterfaces.SmartMotor;
+import org.jetbrains.annotations.NotNull;
+
+public class SpitterClass extends SubsystemBase {
```
Comment (ysthakur) [inline src/main/java/frc/team449/_2022robot/cargo/spitterClass.java:7]: Just `Spitter` will do, no need for a `Class` suffix

File: src/main/java/frc/team449/_2022robot/cargo/spitterClass.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package frc.team449._2022robot.cargo;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team449.generalInterfaces.SmartMotor;
```
Comment (ysthakur) [inline src/main/java/frc/team449/_2022robot/cargo/spitterClass.java]: We're not using `SmartMotor` anymore
```suggestion
```

File: src/main/java/frc/team449/_2022robot/cargo/spitterClass.java (hunk 1 above)
Comment (ysthakur) [inline src/main/java/frc/team449/_2022robot/cargo/spitterClass.java]: Just `Spitter` will do, no need for the `Class` suffix
```suggestion
public class Spitter extends SubsystemBase {
```

File: src/main/java/frc/team449/_2022robot/cargo/spitterClass.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import org.jetbrains.annotations.NotNull;
+
+public class SpitterClass extends SubsystemBase {
+
+  #define motor/solenoid/etc. we will be using
```
Comment (ysthakur) [inline src/main/java/frc/team449/_2022robot/cargo/spitterClass.java]:
```suggestion
  //define motor/solenoid/etc. we will be using
```

**Review by ysthakur — CHANGES_REQUESTED** [review]

File: src/main/java/frc/team449/_2022robot/cargo/spitterClass.java (hunk 3 above)
Comment (sirswagger21) [reply src/main/java/frc/team449/_2022robot/cargo/spitterClass.java]: Been doing a lot of Python recently lol

**Review by ysthakur — APPROVED** [review]

**Pull request merged**